index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
```

Comment:
I don't think that's the way to go. It's better to define a single type of object, a "platform object", which always has a given set of internal methods, instead of swapping out different types of objects. That's what we do for all the other internal methods: branch on whether they support named/indexed properties.

If you're concerned about forward-compatibility, you can fall back to [OrdinaryPreventExtensions](https://tc39.github.io/ecma262/#sec-ordinarypreventextensions) for the other cases.